spec/validators/postcode_validator_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "spec_helper"
 
-class SomeModelWithPostcode
+class Location
```

Comment:
I think this is a bit confusing, because the `Location` class now exists, and it isn't using any validations. 

So I think either leave it as it was, so it obviously a fake class for the purposes of testing. Or make a new test for the `LocationRequest` class, which does use this validator.

Reply:
I agree 👍